core/regression_test/regression_test.jl

```diff
@@ -199,7 +199,7 @@ end
     flow_bench = Arrow.Table(flow_bytes_bench)
     basin_bench = Arrow.Table(basin_bytes_bench)
 
-    solver_list = ["QNDF"]
+    solver_list = []
```

Comment:
Maybe just put change all `@test` to [`@test_broken`](https://docs.julialang.org/en/v1/stdlib/Test/#Broken-Tests)?